etc/ci/upload_nightly.sh

```diff
@@ -12,7 +12,12 @@ usage() {
 
 
 upload() {
-    s3cmd put "${2}" "s3://servo-builds/nightly/${1}/"
+    local nightly_upload_dir package_filename package_upload_path
```

Comment:
Just kidding, that won't work for `package_filename`, but it does work for `package_upload_path`.